opm/simulators/flow/FIBlackoilModel.hpp

```diff
@@ -59,23 +59,39 @@ class FIBlackOilModel : public BlackOilModel<TypeTag>
     FIBlackOilModel(Simulator& simulator)
         : BlackOilModel<TypeTag>(simulator)
     {
+#ifdef _OPENMP
+        const auto& gv = this->gridView_;
+        const auto num_elements = gv.size(0);
+        const int nt = omp_get_num_threads();
```

Comment:
Got this error:

```
/home/jakob/code/opm-clean/src/opm-simulators/opm/simulators/flow/FIBlackoilModel.hpp:65:24: error: there are no arguments to ‘omp_get_num_threads’ that depend on a template parameter, so a declaration of ‘omp_get_num_threads’ must be available [-fpermissive]
   65 |         const int nt = omp_get_num_threads();
   ```

You either need to include the OpenMP header:

```cpp
#ifdef _OPENMP
#include <omp.h>
#endif
```
Or use the number of threads set from the parameters:

```cpp
const int nt = Parameters::Get<Parameters::ThreadsPerProcess>();
```

Reply:
Will fix.